predicators/nsrt_learning/strips_learning/__init__.py

```diff
@@ -22,14 +22,12 @@
             importlib.import_module(f"{__name__}.{module_name}")
 
 
-def learn_strips_operators(
-    trajectories: List[LowLevelTrajectory],
-    train_tasks: List[Task],
-    predicates: Set[Predicate],
-    segmented_trajs: List[List[Segment]],
-    verify_harmlessness: bool,
-    verbose: bool = True,
-) -> List[PNAD]:
+def learn_strips_operators(trajectories: List[LowLevelTrajectory],
+                           train_tasks: List[Task],
+                           predicates: Set[Predicate],
+                           segmented_trajs: List[List[Segment]],
+                           verify_harmlessness: bool,
+                           verbose: bool = True) -> List[PNAD]:
```

Comment:
don't forget to `yapf`, I'm guessing this function will not change from master after that